src/plugins/github/githubGitGateway.test.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import {makeRepoId} from "../../core/repoId";
+import {GithubGitGateway} from "./githubGitGateway";
+
+describe("src/plugins/github/githubGitGateway", () => {
+  describe("commitUrl", () => {
+    it("works for a simple example", () => {
+      const repoId = makeRepoId("sourcecred", "example-github");
+      const hash = "0123456adef";
+      const url = new GithubGitGateway().commitUrl(repoId, hash);
+      expect(url).toMatchInlineSnapshot(
+        `"https://github.com/sourcecred/example-github/commit/0123456adef"`
```

Comment:
It would be nice to use an actual commit hash here, so that we can
simply the link to check that it is valid. Say: `ec91adb718a6`.